src/Common/src/Interop/Unix/Interop.Errors.cs

```diff
@@ -152,12 +152,12 @@ internal partial class Sys
     {
         internal static Error GetLastError()
         {
-            return ConvertErrorPlatformToPal(Marshal.GetLastWin32Error());
+            return ConvertErrorPlatformToPal(PInvokeMarshal.GetLastWin32Error());
```

Comment:
It is pretty unfortunate that we are changing these interop wrappers for this - we want them to be identical between CoreCLR, CoreRT and CoreFX. The PInvokeMarshal is looking less and less as a good idea. I think we should just add back the internal Marshal to CoreLib for now instead to avoid the cascading effect and keep these wrappers to be the same.